code/modules/urist/modules/newtrading/NPC/NPC_types/simple_traders.dm

```diff
@@ -167,3 +184,122 @@
 		/obj/item/clothing/mask/breath/medical,\
 		/obj/item/clothing/mask/surgical)
 	mask_chance = 50
+
+//clothing traders
+
+/mob/living/simple_animal/passive/npc/colonist/trader/casual_clothes
+	name = "Casual Clothes Trader"
+	npc_job_title = "Casual Clothes Trader"
+	desc = "A human from one of Earth's diverse cultures. They are a trader who sells regular clothes."
+	trade_categories_by_name = list("casual clothes")
+	jumpsuits = list(\
+		/obj/item/clothing/under/urist/casual/suspenders,\
+		/obj/item/clothing/under/blazer,\
+		/obj/item/clothing/under/det,\
```

Comment:
Personal gripe. These backslashes don't need to be there and just annoy me lmao

Reply:
should i change this for the whole file? cause those are in glloyd's code too